### ISSUE TITLE
Dashboard no longer exposes itself on public hostname

### ISSUE THREAD

**Issue by chris-eaton** [OrleansContrib/OrleansDashboard#66]
After updating from version 1.4x to 1.5 the Dashboard no longer exposes itself on the public IP / Hostname.

It would be good if it either 
[a] exposed itself automatically or
[b] allowed programmatic options to set hostname / ip

**Comment by richorama** [comment]
@enewnham excellent, thanks for suggesting this. Do you propose that `hostname` could come from a setting? If the setting is not set, default to `http://*:{port}`?

**Comment by enewnham** [comment]
Yeah, unfortunately without making breaking changes, your options are limited.

`RegisterDashboard(this GlobalConfiguration config, int port = 8080, string username = null, string password = null, string hostname = null)`

But hey, its alpha software, so you could flip it on its head, and go with some sort of fluent configuration 

```
RegisterDashboard()
    .UseCredentials( string username, string password )
    .UseUrls( params string[] urls );
```

**Comment by richorama** [comment]
@enewnham I have gone with the simple solution for now. However, I like the fluent idea. Feel free to create a PR if you feel inclined to add it :¬)
